src/object/srv_obj.c

```diff
@@ -483,6 +483,8 @@ bulk_transfer_sgl(daos_handle_t ioh, crt_rpc_t *rpc, crt_bulk_t remote_bulk,
 
 		/* Give cart progress a chance to complete some in-flight bulk transfers */
 		if (bulk_iovs >= MAX_BULK_IOVS) {
+			D_WARN("Bulk transfer for %d/%d IOVs\n", bulk_iovs, total_iovs);
```

Comment:
again this is not a warning that should be printed and will potentially fill server logs.

Reply:
Ok, then let me try to reproduce the issue with this PR.